config/samples/core_v1beta1_openstackcontrolplane.yaml

```diff
@@ -57,9 +57,13 @@ spec:
       secret: osp-secret
   glance:
     template:
+      secret: osp-secret
       databaseInstance: openstack
       storageClass: ""
       storageRequest: 10G
+      glanceAPI:
+        type: single
```

Comment:
@abays given we're using statefulset where each instance has stable access to a given pvc, we need to have a "single" instance when "file" is used as a backend